components/sidebar/topLevelNavItems.jsx

```diff
@@ -33,7 +33,7 @@ var MENU_ENTRIES = [
         to: '/clubs'
       },
       {
-        name: "Guides & Resources",
+        name: "guides_and_resources",
```

Comment:
This has already been merged in from another PR, it was a random warning that I noticed while writing this patch.